userdocs/profiles.dev/docs/catalog-docs/add-profiles.md

```diff
@@ -0,0 +1,104 @@
+---
+sidebar_position: 1
+---
+
+# Adding profiles
+
+:::info
+Please refer to the [Introduction](/docs/introduction#catalog) for a list of terms
+around Catalogs.
+:::
+
+To add profiles to a catalog, you must first deploy the catalog API
+and `ProfileCatalogSource` controller to your cluster.
+
+This can be done with the `pctl install` command. By default, this command
```

Comment:
```suggestion
This can be done with the `pctl prepare` command. By default, this command
```

honestly do you even work here? :trollface:

Reply:
Whatever do you mean? Do you think I am running some elaborate con in which I pretend to be a weave employee as part of some obscure and devious plan for world domination in which the profiles project is a small but significant piece? I have never been so offended in my HELP JAKE DON'T LISTEN THAT IS NOT REALLY M life. How very dare you. Now if you don't mind I am off to do Claudia type things that Claudia would do. Indeed. Good day.

Reply:
🤯